src/server/index.js

```diff
@@ -37,30 +36,13 @@ if (process.env.APP_INSIGHTS_KEY) { // eslint-disable-line no-process-env
 const createServer = () => {
     const themeDir = path.join(hexoDir, theme);
     const layoutsDir = path.join(themeDir, 'layout');
-    const partialsDir = path.join(layoutsDir, 'partials');
-
-    /* Helpers path */
-    const helpersPath = path.join(themeDir, 'helper/index.js');
-
-    /* Required Helpers */
-    const miscHelpers = require(helpersPath)();
-
-    const helpers = Object.assign(handlebars.helpers, miscHelpers); // Combine helpers
     const app = express();
 
     app.disable('x-powered-by');
-    app.set('views', partialsDir);
-
-    const hbs = exphbs.create({
-        defaultLayout: 'common',
-        extname: '.hbs',
-        helpers,
-        layoutsDir,
-        partialsDir
-    });
+    app.set('views', layoutsDir);
 
-    app.engine('hbs', hbs.engine);
-    app.set('view engine', 'hbs');
+    app.engine('ejs', consolidate.ejs);
```

Comment:
Look into adding `utils` into `app.locals` so we don't have to pass it all around. Also remember that the `utils` from the formatter doesn't have `hasSubPage` so it will have to be merged.